extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -475,15 +473,18 @@ export class UserNotificationManager {
   }
 }
 
+
 /**
- * @param {!Window} window
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+ * @visibleForTesting
  */
-export function installUserNotificationManager(window) {
-  registerServiceBuilder(window,
-      'userNotificationManager',
-      UserNotificationManager);
+export function installUserNotificationManagerForTesting(ampdoc) {
+  registerServiceBuilderForDoc(ampdoc, SERVICE_ID, UserNotificationManager);
 }
 
-installUserNotificationManager(AMP.win);
 
-AMP.registerElement('amp-user-notification', AmpUserNotification, CSS);
+// Register the extension services.
+AMP.extension(TAG, '0.1', function(AMP) {
```

Comment:
Is there a benefit of making this callback a regular function vs a fat-arrow function?